catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -326,7 +326,7 @@ private Object getCatalogMgrObject(String managerId) {
   @Override
   public LibraryDescription get_library_description() throws ProcessingFault, SystemFault {
     LOGGER.trace("get_library_description called");
-    String host = System.getProperty("org.codice.ddf.system.hostname");
+    String host = System.getProperty("org.codice.ddf.external.hostname");
```

Comment:
❓ Shouldn't this and the one below use the constants in `SystemBaseUrl` instead?

Reply:
Good catch! Was trying to replace all of these! Will update